src/test/cli.up.test.ts

```diff
@@ -25,6 +25,13 @@ describe('Dev Containers CLI', function () {
 	});
 
 	describe('Command up', () => {
+		it('should succeed (dockerfile) with supported --error for poetry', async () => {
```

Comment:
What's the purpose of this test with respect to the PR? 

If you want to test the poetry scenario, then you can create a new dev container config which fails similar to https://github.com/devcontainers/cli/issues/845

Then, validate if you the `poetry: command not found` error. However, in this case it's alright to not add the tests.

Reply:
test removed